examples/gaussians.c

```diff
@@ -0,0 +1,740 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
```

Comment:
Removing all the "#if 0"s here would be good also.

Reply:
Done